collections/_infographics/energy/electricity-per-capita-world.md

```diff
@@ -14,4 +14,29 @@ data-orig:
+* Globally, electricity production increased by 75% between 2000 and 2019, while at the same time, the population has grown by 1.6 billion people. n per capita terms, electricity generation increased by 39%, and in absolute terms, there was an increase in the capacity of all electricity generation sources except nuclear. The share of coal in electricity production fell slightly (from 37.9% to 35.2%), and the percentage of nuclear power fell even more significantly (from 16.9% to 10.5%). In contrast, there was an increase in the share of natural gas (from 18.2 % to 23.5%) and renewable sources (from 19.5% to 27.3% due to the development of wind and solar power plants).
+* **Trends of individual regions** observed from the graph:
+  * In the **European Union**, electricity consumption is virtually unchanged per capita, but there is a significant substitution of coal for renewables in electricity generation.
+  * Electricity consumption per person in the **USA** has fallen slightly but is still many times higher than the world average. Since 2007, there has been a decrease in the share of coal in electricity production, which is, however, being replaced mainly by natural gas, partly also by renewable sources.
+  * In **China**, the per capita electricity consumption increased fivefold between 2000 and 2019, with average consumption already at 82% of the European Union average. There has been an increase in the capacity of all sources of electricity generation, yet coal is still the dominant source in China, with a 62% share. On a per capita basis, China generated 3,180 kWh from coal in 2019, which is the most of the regions shown. By comparison, the Czech Republic produced 3,439 kWh per person from coal in the same year.
+  * **India** has more than doubled its per capita electricity generation over this period, yet it is less than a third of the global average. Coal is still the dominant source of electricity generation, with a 72% share, while other sources are also gradually developing.
+  * The **Rest of the World** represents a very diverse group, where electricity production per person in some countries exceeds the USA (i.e., in Canada or the oil countries of the Middle East). In contrast, in many other countries, it is lower than in India (usually in African and some Asian countries).
+* The chart shows the electricity produced in each region, not consumed. For example, in the case of the Czech Republic, net exports make up almost 15% of the electricity produced. However, for the prominent regions in the graph, imports and exports are practically negligible, being below 1% for all regions.
```

Comment:
"negligible, being below 1% for all regions." to "negligible: below 1% for all regions."

Reply:
Updated.